docs/ybm.md

```diff
@@ -17,6 +17,7 @@ ybm [flags]
       --config string      config file (default is $HOME/.ybm-cli.yaml)
       --debug              Use debug mode, same as --logLevel debug
   -h, --help               help for ybm
+      --host string        YBM Api hostname
```

Comment:
uppercase YBM refers to YugabyteDB Managed, lowercase ybm refers to the cli binary. So in this case, what are you referring to? All instances of 'YBM' should be replaced with YugabyteDB Aeon.
What is the YBM Api hostname? What is it for?
Api - > API

Reply:
`host` has now been removed. All occurrences of `YBM` in the docs have been replaced with YugabyteDB Aeon.